CMSgov/mint-app
# [NOREF] Update Dependencies

# NOREF


## Changes and Description

- Updated the following
  - glob
  - regenerator-runtime
  - webpack
  - cypress
  - sass
  - yup
  - autoprefixer
 


## How to test this change

1. `yarn install`
2. `scripts/dev down && scripts/dev up`
3. Verify nothing breaks 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
